content/boards/arduino-mega-2560/index.md

```
@@ -24,3 +24,7 @@
 ## Pinout
 
 {{< pinout >}}
+
+## Additional resources
+
+- [Official technical documentation](https://docs.arduino.cc/hardware/mega-2560/)
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)